pacu/modules/iam__enum_roles/main.py

```diff
@@ -111,7 +112,7 @@ def run(args, role_name, pacu_main, iam):
                 return data
             else:
                 print('  Unhandled error: {}'.format(str(error)))
-                return data
+                raise error
```

Comment:
I don't think there's any reason to return the data here if the module fails, doing so also prevents the error from being logged.